aries-site/src/layouts/content/Example/CompareUseCases.js

```diff
@@ -0,0 +1,55 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { Box, Image, ResponsiveContext, Text } from 'grommet';
+import { Close, Checkmark } from 'grommet-icons';
+
+export const CompareUseCases = ({ imageSrcGood, imageAltGood, captionGood, imageSrcBad, imageAltBad, captionBad }) => {
+
+    const size = useContext(ResponsiveContext);
+    let direction;
+    if (size !== 'small') {
+        direction = 'row';
+    } else direction = 'column';
+
+    let width;
+    if (size !== 'small') {
+        width = 'medium';
+    } else width = '100%';
+
+    return (
+        <Box
+            direction={direction}
+            gap="large"
+            pad={{ vertical: 'medium' }}>
+            <Box width={width}>
+                <Image src={imageSrcGood} alt={imageAltGood} />
```

Comment:
I know this isn't quite as DRY as it could be - I considered abstracting the "Good" and "Bad" out into their own components, but in the end, felt it wasn't a real value add to add an additional abstraction for some slight redundancy. Also, I'd need to add a conditional statement for the icon choice. Thoughts?

Reply:
This approach is fine, but I tend to view this as a single component which then can be reused:

```
const DoDontExample = ({ 
    children, // can be a React element, node, image, .... anything really
    description, // string
    doDont, // poor naming here, but providing concept string which could be one of ['do', 'dont']
}) => {
    return (
        <Box>
            { children }
            <Box direction="row" gap="xxsmall">
                { doDont === 'do' ? 
                    <Checkmark size="medium" color="green!" /> : 
                    <Close size="medium" color="red" />
                 }
                 <Paragraph>{ description }</Paragraph>
            </Box>
        </Box>
    );
};
```

Then the overall implementation would be:
```
<Box
       direction={direction}
       gap="large"
 >
      <DoDontExample doDont="do" description={descriptionDo} >
          // Insert React element, node, <Image />, <Paragraph > anything you want here
      </DoDontExample>
      <DoDontExample doDont="dont" description={descriptionDont} >
          // Insert React element, node, <Image />, <Paragraph > anything you want here
      </DoDontExample>            
 </Box>
 ```